draft-ietf-quic-multipath.md

```diff
@@ -517,16 +517,50 @@ the server MAY wait for a short, limited time such as one PTO if a path
 probing packet is received on a new path before sending the
 CONNECTION_CLOSE frame.
 
+### Idle Timeout {#idle-time-close}
```

Comment:
I moved this section up as this section is supposed to be a subsection of "[Path Close](https://quicwg.org/multipath/draft-ietf-quic-multipath.html#name-path-close)" while the other sections now afterwards are not supposed be subsections of "Path Close".

The text was only moved; no text changes!